src/components/CityList.js

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { FlatList, View, StyleSheet } from 'react-native';
-import * as cityData from '../service/cityData';
+import * as city from '../services/city';
 import CityListItem from './CityListItem';
 
-function CityList() {
-  return (
-    <View style={style.list}>
-      <FlatList data={cityData.cityList} renderItem={({ item }) => <CityListItem city={item} />} />
-    </View>
-  );
+class CityList extends Component {
+  renderItem = ({ item }) => <CityListItem city={item} />;
```

Comment:
use autobind & ordinary function.

P.s. here is that article I promised you about performances of arrow function compared to "ordinary" functions in react components: https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1.